rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/toolWindow/log/UnityLogPanelToolbarBuilder.kt

```diff
@@ -64,34 +65,18 @@ object UnityLogPanelToolbarBuilder {
             }
         }
 
-        fun createBeforePlay() = object : ToggleAction("Show/Hide messages before last Play", "", null) {
+        fun createBeforePlay() = object : ToggleAction("Messages before last Play in Unity", "", UnityIcons.LogView.FilterBeforePlay) {
             override fun isSelected(e: AnActionEvent) = model.timeFilters.getShouldBeShownBeforePlay()
             override fun setSelected(e: AnActionEvent, value: Boolean) {
                 model.timeFilters.setShowBeforePlay(value)
             }
-
-            override fun update(e: AnActionEvent) {
-                if (isSelected(e))
-                    e.presentation.text = "Hide messages before last Play"
-                else
-                    e.presentation.text = "Show messages before last Play"
-                super.update(e)
-            }
         }
 
-        fun createBeforeInit() = object : ToggleAction("Show/Hide messages before last [InitializeOnLoad] execution", "", null) {
+        fun createBeforeInit() = object : ToggleAction("Messages before last [InitializeOnLoad] execution", "",  UnityIcons.LogView.FilterBeforeRefresh) {
```

Comment:
Is there a better message than this? It's very technical. Does it mean last AppDomain reload?

Reply:
Yes

Reply:
How about "Messages since last AppDomain reload"? While it's still a bit technical, everyone knows what AppDomain reloads are because they happen so often